meshapp/src/profile/profile-page.tsx

```diff
@@ -0,0 +1,332 @@
+import React from "react";
+import {
+  Box,
+  Grid,
+  Typography,
+  createTheme,
+  ThemeProvider,
+  Chip,
+} from "@mui/material";
+
+import ProfileTextField from "./profile-textfield";
+import { Profile } from "./types/profile";
+import "./styling/profile-page.css";
+
+const theme = createTheme({
+  palette: {
+    mode: "light",
+    primary: {
+      main: "#0b7d66",
+    },
+  },
+  typography: {
+    h1: {
+      fontFamily: "cocogoose",
```

Comment:
Would it be possible to group the h1-h4 styles into a single common style using a CSS class name?

Reply:
yep, done 👍